.github/workflows/macos.yml

```diff
@@ -23,11 +23,7 @@ jobs:
           python-version: '3.12'
       - name: Install dependencies
         run: |
-          brew install bison graphviz libxml2 meson ninja pyyaml
-          # HACK: force Doxygen version 1.9.6, because 1.9.7 breaks our cool URIs.
-          curl -O https://raw.githubusercontent.com/Homebrew/homebrew-core/d2267b9f2ad247bc9c8273eb755b39566a474a70/Formula/doxygen.rb
-          brew install --formula doxygen.rb
-          rm doxygen.rb
```

Comment:
fwiw, I think an option here would also be to simply disable the doc build on macos if this ever happens again. I don't think we really need to CI that, especially if it's a bit buggy.

Come to think of it: maybe we should just disable the documentation build by default? I doubt there are (m)any users out there who need a local doxygen result -> #471

Reply:
#471 merged. We can remove Doxygen requirement here

Reply:
👍